src/components/Dialogs/FileBrowser/FileList/FileListComponent.tsx

```diff
@@ -31,12 +31,12 @@ export class FileListComponent extends React.Component<{
         const fileEntries = [];
         const fileList = this.props.files;
         if (fileList) {
-            let sortedDirectories: string[] = [];
-            if (fileList.subdirectories && fileList.subdirectories.length) {
+            let sortedDirectories = fileList.subdirectories.slice();
```

Comment:
this will re-introduce the previous bug, because `fileList.subdirectories` could be undefined (which is why the line above has a check)